Views/Home/Index.cshtml

```diff
@@ -2,7 +2,49 @@
     ViewData["Title"] = "Home Page";
 }
 
-<div class="max-w-md m-auto px-3">
-    <h2 class="text-grey-darker uppercase tracking-wide text-center">@ViewData["Title"]</h2>
-    <p class="mt-5 text-grey-darker">Lorem ipsum dolor sit amet, cum te tantas recteque persecuti, sumo appareat torquatos ei sea, alienum insolens cu mei. Oportere scriptorem ea per. Quo ne civibus ancillae, qui doctus iriure ex, eam suas sanctus vivendum an. Dolores offendit assentior at duo, ex brute quaerendum accommodare qui. Inciderint dissentiet vis id, amet offendit ut sed.</p>
-</div>
\ No newline at end of file
+<!-- Refrence for image postioning  -->
+ <!-- https://stackoverflow.com/questions/67081460/how-can-i-align-image-horizontally-in-tailwind-css -->
```

Comment:
Same as comment above. Then we're all set to merge the PR

Reply:
Ok comments have been removed